api/migrations/0004_fixAlbumThingContraint.py

```diff
@@ -1,3 +1,5 @@
+api/migrations/0004_fixAlbumThingContraint.py 
+
```

Comment:
I forget it always too, but the name scheme in python is always with underscores. Please rename the file to 0004_fix_album_thing_constraint.py